config/csp/google_supported_domains.csv

```diff
@@ -0,0 +1,190 @@
+.google.com
```

Comment:
I believe #1650 added these domains as an attempt to solve the analytics issue we had few weeks ago? If I recall correctly, the issue was fixed and it was unrelated to the introduced changes, so maybe we should clean them up?

Reply:
It may not have caused that particular issue but we still had CSP violations against some of these domains; I think given we can't control which domains Google are using (and it probably changes now and then) we should leave them in to be safe

Reply:
ah ok then :)